util/registry_utils.ts

```diff
@@ -1,4 +1,5 @@
 /* Copyright 2020 the Deno authors. All rights reserved. MIT license. */
+import semver from "semver";
```

Comment:
As I said in my other comment, the website is served from Deno Deploy, so you can import from `deno.land/x/semver` just as you were in the original version. You also don't have to add `semver` to the package.json because of that.

Reply:
Okay. I updated it.